client/src/components/App/index.js

```diff
@@ -1,14 +1,33 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Login from '../utils/Login/index';
 import Error404 from '../utils/Error404/index';
 import './style.css';
 
-export default () => {
-  return (
-    <Router>
-      <Switch>
-        <Route exact path="*" component={Error404} />
-      </Switch>
-    </Router>
-  );
-};
+export default class App extends React.Component {
+  state = {
+    tableNumber: null,
+  };
+
+  updateTableNumber = e => {
+    this.setState({ tableNumber: e });
+  };
+
+  render() {
+    return (
+      <Router>
+        <Switch>
+          <Route
+            exact
+            path="/login"
+            component={props => (
```

Comment:
If you're using an inline function for rendering use `render`
[react-router-dom `Route/component`](https://reacttraining.com/react-router/web/api/Route/component)
[react-router-dom `Route/render`](https://reacttraining.com/react-router/web/api/Route/render-func)
```suggestion
            render={props => (
```